src/main/java/forus/naviforyou/domain/realTime/service/BusService.java

```diff
@@ -0,0 +1,214 @@
+package forus.naviforyou.domain.realTime.service;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import forus.naviforyou.domain.findRoute.dto.response.TravelRouteRes;
+import forus.naviforyou.domain.realTime.dto.request.BusStationReq;
+import forus.naviforyou.domain.realTime.dto.response.ItemList;
+import forus.naviforyou.domain.realTime.dto.response.ServiceResult;
+import forus.naviforyou.global.error.dto.ErrorCode;
+import forus.naviforyou.global.error.exception.BaseException;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.StringReader;
+import java.net.URL;
+import java.net.URLEncoder;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.XML;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.net.HttpURLConnection;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Service
+public class BusService {
+
+    @Getter
+    @Value("${realtime.serviceKey}")
+    private String serviceKey;
+
+    public List<ItemList> stationInfo(BusStationReq busStationReq , Boolean lowStation){
+
+        double doubleX = Double.parseDouble(busStationReq.getX());
+        double doubleY = Double.parseDouble(busStationReq.getY());
+        String stationId = getStationByName(doubleX,doubleY);
+
+
+        List<ItemList> stationInfoRes;
+        if(lowStation){
+            stationInfoRes = getLowStationByUidItem(stationId);
+        }
+        else{
+            stationInfoRes = getStationByUidItem(stationId);
+        }
+        return stationInfoRes;
```

Comment:
```suggestion
        if(lowStation){
            return getLowStationByUidItem(stationId);
        }
       return getStationByUidItem(stationId);
```
if-else 문으로 쓰는 것보다는 이런 식으로 if문에서 return 하는 게 저는 더 선호하는 방식인데 이런 방식은 어떠신가요?

Reply:
이것도 좋은 방법 같습니다. 말씀하신 내용 반영하여 수정하였습니다!